### ISSUE TITLE
Adding additional fonts

### ISSUE THREAD

**Issue by 1mzino** [quilljs/quill#3658]
I am using SvelteJS with vite to quickly get a static site running which hosts its own rich-text editor.. I can get Quill running just fine and the toolbar works.. (even if I am not able to render it in a container rather than document.body)

I have followed the guide on adding fonts and am only able to see "Sans Serif" on the dropdown menu rather than my own fonts..

I am expecting to see 'Austin News' be a selectable font.. perhaps I am missing some steps? FYI I also have an external stylesheet with the 'Austin News' fontface

```
<script lang="ts">
  import { onMount } from "svelte";
  let quill;

  onMount(async () => {
    const { default: Quill } = await import("quill");
    let fonts = ["Austin News"];
    var FontAttributor = Quill.import("attributors/class/font");
    FontAttributor.whitelist = fonts;
    Quill.register(FontAttributor, true);

    const toolbarOptions = [
      [{ font: fonts }],
      [{ header: [1, 2, 3, false] }],
      ["bold", "italic", "underline", "strike", { color: ["#F00", "#0F0", "#00F", "#000", "#FFF", "color-picker"] }],
      ["link", "code-block"],
    ];

    quill = new Quill(".editor", {
      modules: {
        toolbar: toolbarOptions,
      },
      theme: "snow",
      placeholder: "Type something..",
    });
  });
</script>
```

**Comment by gretchelin** [comment]
I'm also facing this problem with vue3. 

It seems I can't mix class style with inline style, as using Quill with class style works for font, although I had to hold off calling `new Quill()` until after `nextTick()` or DOM is updated with changes, otherwise, the toolbar will not render. With inline style, the select dropdown only show "Sans Serif" as custom font name, even though the class being applied to the text is correct, no matter if I await for nextTick or not.

Also, as pointed in the guide here: https://quilljs.com/guides/how-to-customize-quill/#customizing-attributors
I had to add custom classes to my page to handle these custom fonts. And another thing to note: since the font name used in the `whitelist` is what is being passed as `ql-font-<fontname>`, I can't use space in the font name.
